src/build.d

```diff
@@ -987,6 +996,38 @@ version(Windows)
         // Could not find or obtain vswhere.exe
         throw new Exception("Could not obtain vswhere.exe. Consider downloading it from https://github.com/Microsoft/vswhere and placing it in your PATH");
     }
+
+    /*
```

Comment:
Nit: This is not DDOC (need at least 2 stars)

Reply:
Actually all functions could use proper DDoc.  I'll address that holistically in a followup PR.